driver-helper/bc125at-perl-driver-helper.c

```diff
@@ -24,6 +24,8 @@
 * OTHER DEALINGS IN THE SOFTWARE.
 */
 
+/* Lines 51-54 Copywright (c) 2022 Max Loiacono, MIT License */
```

Comment:
Minor review notes:
* "Copywright" should be "Copyright" here.
* Re: line numbers - Hypothetically, if anyone made additional code changes (unlikely given the idleness of this project, but still), these line numbers could get out of sync with the source code. If you prefer in the future, you're welcome to add your copyright line directly below mine without specifying exact line numbers.

Thanks for the contribution. I can merge as-is for now.

Reply:
How embarrassing... I did not notice the spelling mistake initially. I do not necessarily care about attribution for this change; I just wanted the program to work :)

If you'd like to just edit or remove the copyright line, I'd have no problem!

Reply:
No problem. I went ahead and made an edit just to fix the typo.

Thanks again for the fix